src/app/resources/view-resources/resources-view.component.ts

```diff
@@ -17,7 +17,62 @@ import { Subject } from 'rxjs/Subject';
       height: 80vh;
       border: none;
     }
-  ` ]
+    .resRating {
+      display: grid;
+      max-width: 230px;
+      margin-left: auto;
+    }
+    .rating {
+      text-align: center;
+    }
+    $gray: #aaa;
+
+    body{
+      background: black;
+      color: white;
+    }
+    .container{

```

Comment:
Please stick to Material components and basic styling until we have more specific designs from UX